core/commonTest/src/kotlinx/serialization/WrappedSerialDescriptorTest.kt

```diff
@@ -0,0 +1,70 @@
+package kotlinx.serialization
+
+import kotlinx.serialization.builtins.ListSerializer
+import kotlinx.serialization.builtins.MapSerializer
+import kotlinx.serialization.builtins.serializer
```

Comment:
We usually prefer star imports in code style

Reply:
Sorry, IntelliJ automatically added those. I'll fix it.